KPP/fullchem/fullchem_RateLawFuncs.F90

```diff
@@ -2570,24 +2570,32 @@ FUNCTION IONO2uptkByH2O( H ) RESULT( k )
     k    = 0.0_dp
     srMw = SR_MW(ind_IONO2)
     !
-    ! Tropopsheric sulfate (use T-dependent gamma, cf Deiber et al 2004)
-    area  = H%ClearFr * H%xArea(SUL)
-    gamma = MAX( ( 0.0021_dp * TEMP - 0.561_dp ), 0.0_dp )
-    k     = k + Ars_L1K( area, H%xRadi(SUL), gamma, srMw )
-    !
```

Comment:
I think we still need to keep IONO2 uptake on tropospheric sulfate aerosols in IONO2uptkByH2O. What do you think, @beckyalexander?

Reply:
Thanks @viral211. This is now restored in #1899.